docs/topic-guides/custom-pod-specs.txt

```diff
@@ -17,11 +17,23 @@ use Determined on Kubernetes.
 
 In this topic guide, we will cover:
 
+#. How Determined uses pod specs.
 #. The different ways to configure custom pod specs.
 #. Supported pod spec fields.
 #. How to configuring default pod specs.
 #. How to configuring per-task pods specs.
 
+*****************************
+How Determined Uses Pod Specs
+*****************************
+
+All Determined tasks are launched as pods. Determined pods consists of an
+initContainer named ``determined-init-container`` and a container named
```

Comment:
I mean since you have already explained `determined-container` it seems a bit odd not to explain `determined-init-container`. How about just saying both of them are used for executing the workload?